tools/debugging/README.md

```diff
@@ -73,6 +73,17 @@ $ GST_DEBUG="GST_TRACER:7" GST_TRACERS="leaks(GstEvent,GstMessage)" \
 gst-launch-1.0 videotestsrc num-buffers=10 ! fakesink
 ```
 
+#### Case study 2: Tracing individual NNStreamer elements with GST_DEBUG and silent property
+
+Each NNStreamer elements has `silent` property. An NNStreamer element can be traced in verbose mode by setting `FALSE` to `silent` property.
+
+```bash
+$ GST_DEBUG="tensor_decoder:7" \
+gst-launch-1.0 fakesink ! tensor_decoder silent=FALSE ! fakesrc
+```
+
+In each element's source code, there is `DEFAULT_SILENT` macro that allows you change default `silent` value. Setting `FALSE` to `DEFAULT_SILENT` and rebuilding the library will set verbose mode of the element by default without changing your application code.
```

Comment:
The content is good for me. I like to update as follows:
* ... allows you to change the default `silent` value. ...

Reply:
I fixed it, thanks :)